### PR TITLE
docs: updated documentation to display on validation message

### PR DESCRIPTION
**Pull request by laraharrow**
Thank so much for contributing to @looker/components!

The CI & [README](https://github.com/looker-open-source/components/blob/main/README.md) should help to make most of our code standards clear

Before requesting review from another developer please delete the lines above and summarize your change.

### Requirements

Please check the following items are addressed in your pull request (or are not applicable)

- [ ] a11y impact (FUTURE: Make aXe pass req'd for CI ✅)

#### Image snapshots (choose one)

  - [ ] yes
  - [x] not applicable

#### Documentation updated (choose one)

  - [x] yes
  - [ ] not applicable

### CODE REVIEW

**Comment by laraharrow** [comment]
there was a question on ticket [1598](https://looker.atlassian.net/browse/LENS-1598) about the validation not being triggered at first render.
I believe that it shouldn't happen as it should be triggered when it get focus and the behavior passed is validated.

File: www/src/documentation/components/fields/field-text.mdx [hunk 1]
```diff
@@ -18,7 +18,7 @@ The `<FieldText />` component is composed of an `<InputText />` component and a
 
 The `<FieldText />` component accepts `validationMessage` as a property. This is an object with properties `type` and `message`.
 
-See `ValidationMessage` [here](/#!/ValidationMessage)
+See `ValidationMessage` [here](https://github.com/looker-open-source/components/blob/main/packages/components/src/Form/ValidationMessage/ValidationMessage.tsx)
```
Comment (dbchristopher) [inline www/src/documentation/components/fields/field-text.mdx:21]: What is the intent of linking to the component source here? Honestly I think you could just delete this line, as I don't think it clarifies anything

**Review by dbchristopher — CHANGES_REQUESTED** [review]
Looks like `components/forms/demos/index.ts` needs to be deleted now. Gatsby won't start because it's still trying to export that removed demo:

Screen Shot 2021-02-19 at 10 30 07 AM

Good call moving the demo source inline though!

File: www/src/documentation/components/fields/field-text.mdx (hunk 1 above)
Comment (laraharrow) [reply www/src/documentation/components/fields/field-text.mdx:21]: So yeah that is not the final solution. But it solves the problem of a broken link. I'm going to create a new ticket to specifically create a docs with the appropriated documentation to `ValidationMessage`.

Comment (dbchristopher) [reply www/src/documentation/components/fields/field-text.mdx:21]: works for me!

**Review by dbchristopher — APPROVED** [review]

**Pull request merged**